### PR TITLE
Add gpu benchmark examples for other models.

### PR DESCRIPTION
**Pull request by xingyu-long**

### CODE REVIEW

File: tensorflow/python/keras/benchmarks/keras_examples_benchmarks/mnist_irnn_benchmark_test.py [hunk 1]
```diff
@@ -112,6 +112,26 @@ def benchmark_irnn_mnist_bs_1024(self):
     self.report_benchmark(
         iters=run_iters, wall_time=wall_time, metrics=metrics, extras=extras)
 
+  def benchmark_irnn_mnist_bs_1024_gpu_1(self):
```
Comment (yhliang2018) [inline tensorflow/python/keras/benchmarks/keras_examples_benchmarks/mnist_irnn_benchmark_test.py:115]: Any reason to use `gpu_1` here?

File: tensorflow/python/keras/benchmarks/keras_examples_benchmarks/reuters_mlp_benchmark_test.py [hunk 2]
```diff
@@ -78,6 +78,27 @@ def benchmark_mlp_reuters_bs_128(self):
     self.report_benchmark(
         iters=run_iters, wall_time=wall_time, metrics=metrics, extras=extras)
 
+  def benchmark_mlp_reuters_bs_128_gpu_1(self):
```
Comment (yhliang2018) [inline tensorflow/python/keras/benchmarks/keras_examples_benchmarks/reuters_mlp_benchmark_test.py:81]: Ditto as above.

File: tensorflow/python/keras/benchmarks/keras_examples_benchmarks/mnist_irnn_benchmark_test.py (hunk 1 above)
Comment (xingyu-long) [reply tensorflow/python/keras/benchmarks/keras_examples_benchmarks/mnist_irnn_benchmark_test.py:115]: I would like to provide a different GPU benchmark in each model, not only gpu_2 in here. And the results should be fine.

Comment (yhliang2018) [reply tensorflow/python/keras/benchmarks/keras_examples_benchmarks/mnist_irnn_benchmark_test.py:115]: Yeah, with that said, could you try 3 or 4 GPUs? As 1 gpu is not really distributed training, here the goal is to measure dist-strat.

Comment (xingyu-long) [reply tensorflow/python/keras/benchmarks/keras_examples_benchmarks/mnist_irnn_benchmark_test.py:115]: Sounds great, I will change these two to use 3 or 4 gpus.

**Review by yhliang2018 — APPROVED** [review]

**Pull request merged**